smokes/e2e/home.scenarios.coffee

```diff
@@ -0,0 +1,42 @@
+# coffee script
+# test goal: checks the the number of element present in home page
+# to test this part: two different builds need to be started
+
+
+forcePage = require('./force.coffee')
+builderPage = require('./builder.coffee')
+homePage = require('./home.coffee')
+
+
+describe('', () ->
+    force = null
+    builder = null
+    home = null
+
+    beforeEach(() ->
+        builder = new builderPage('runtest', 'force')
+        force =  new forcePage()
+        home = new homePage()
+        builder.goDefault()
+    )
+
+    describe 'manage home web page', () ->
+        it 'should go to the home page and check the different builder', () ->
+            researchData = {
+                "0" : "slowruntest",
+                "1" : "runtests"
+            }
+            #runtest = "runtests"
+            #slowtest =
+            builder.go()
+            builder.goForce()
+            force.getStartButton().click()
+            #browser.get('#/builders/1/builds/15')
+            home.go()
+            alltitles = element.all(By.css(".panel-title"))
```

Comment:
we probably want to abstract those in homePage.getPanel(0), homePage.getPanel(1)